src/app.rs

```diff
@@ -340,6 +344,13 @@ lazy_static! {
         doc!(h, "debug",
              "Show debug messages.",
              "Show debug messages. Please use this when filing a bug report.");
+        doc!(h, "dfa-size-limit",
+             "The upper size limit of the generated dfa.",
+             "The upper size limit of the generated dfa. The default limit is \
+              10Mb. This should only be changed on very large regex inputs \
```

Comment:
I think this should say `10M` instead of `10Mb`. (Even so, the `Mb` suffix means "megabit", which is otherwise wrong. :-))